content/en/service_management/incident_management/investigate/_index.md

```diff
@@ -0,0 +1,81 @@
+
+After an incident is resolved, an option appears in the header to generate a postmortem Notebook using a [postmortem template][4]. Configure your postmortem templates in the app; navigate to the [Incident Settings][5] page to predefine the structure and content of your postmortems.
+
+### Overview tab
+
+The Overview tab serves as the main page to view an incident's properties and define customer impact. By default, it includes properties such as Root Cause, Services, Teams, Detection Method, and Summary, categorized into the What Happened, Why it Happened, and Attributes sections. 
+
+Add more property fields using `<KEY>:<VALUE>` pairs from Datadog metric tags, or create custom fields through [Incident Settings][6]. Assign values to these properties to improve search and query in the Incident Homepage and Incident Management Analytics. To prioritize critical information, property fields can be reordered and moved under various headings. 
+
+For customer-facing incidents, specify impact details by adding them in the **Impacts** section:
+
+{{< img src="/service_management/incidents/investigate/incident_details_impacts.png" alt="Your image description" style="width:90%;" >}}
+
+1. Click **Add**.  
+2. Specify a start date and time for the impact.  
+3. Specify an end date and time for the impact or leave blank if the impact is still ongoing.  
+4. Describe the nature of the impact on customers in `Scope of impact`.  
+5. Click **Save**.
+
+In addition to housing your property fields, the Overview tab also provides the following at-a-glance summary modules:
+
+| Summary Module | Description |
+|-------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Condensed Timeline | Displays the times when the incident changes state as well as when impact started and ended for a high-level view of the incident's lifecycle. |
+| Latest Notifications| Displays the most recent notification sent for the incident, with quick access to the full list of notifications in the [Notification tab][7].|
+| Pending Tasks | Displays the most recent incomplete task, with quick access to the full list of tasks in the Remediation tab. |
+| Responders | Displays the current incident commander and avatars for the remaining responders assigned to the incident. |
+| Recent Timeline Entries | Displays the five most recent entries in the incident timeline, with quick access to see the entire Timeline tab. |
+
+## Additional investigation tools
+
+After declaring an incident, responders can utilize the Incident Details Page to apply available information to describe and analyze the incident thoroughly. 
```

Comment:
```suggestion
After declaring an incident, responders can use the Incident Details Page to apply available information so that they can describe and analyze the incident thoroughly. 
```